packages/components/slider/src/use-slider-thumb.ts

```diff
@@ -101,16 +109,19 @@ export function useSliderThumb(props: UseSliderThumbProps) {
   };
 
   const getTooltipProps = () => {
-    const value = state.values[index ?? 0];
+    const value = numberFormatter
+      ? numberFormatter.format(state.values[index ?? 0])
+      : state.values[index ?? 0];
 
     const content =
       renderOutput && typeof renderOutput === "function" ? renderOutput(`${value}`) : value;
 
     return {
       ...tooltipProps,
-      content,
-      isOpen: isHovered || isDragging,
-      placement: isVertical ? "right" : "top",
+      placement: tooltipProps?.placement ?? isVertical ? "right" : "top",
```

Comment:
Missing curly braces?
```suggestion
      placement: tooltipProps?.placement ?? (isVertical ? "right" : "top"),
```

Reply:
@jguddas curly braces are automatically added/removed by eslint/prettier, in this is not needed because the first validation is not a ternary